x-pack/plugins/security_solution/public/management/components/endpoint_responder/types.ts

```diff
@@ -15,3 +16,17 @@ export interface EndpointCommandDefinitionMeta {
 export type EndpointResponderExtensionComponentProps = ManagedConsoleExtensionComponentProps<{
   endpoint: HostMetadata;
 }>;
+
+export interface ActionRequest {
```

Comment:
Yeah, best to rename this. Its confusing (IMO 😀 )

Reply:
3389576eebb1083b6d591c355fd6a8716eef9dfb